packages/matter.js/src/MatterDevice.ts

```diff
@@ -281,11 +281,17 @@ export class MatterDevice {
             // TODO Change when refactoring MatterDevice away
             this.initialCommissioningCallback();
         }
+        this.sendFabricAnnouncements([fabric], true).catch(error =>
```

Comment:
I also thought but in fact cancelling a Promise is not possible. so wouod be only the await case, but on shutdown this is also not too relevant - especially if the promise "hangs". In fact on shutdown e close all underlaying network connections, so the "catch" case should be called because exceptions would happen. Thats why I decided to just go that way. 

If you have a better idea, please tell.

Reply:
This is along the lines of below, resolving as well